### ISSUE TITLE
Titanic - Machine Learning from Disaster

### ISSUE THREAD

**Issue by harshita9621** [ZeusCodes-Official/DS-Olympus-#44]
## About this file

- These predictions assume only female passengers survive.

### The data has been split into two groups: 

- training set (train.csv)
- test set (test.csv)

### Dataset

[https://www.kaggle.com/c/titanic/data](url)

**Comment by harshita9621** [comment]
Can I work on this issue.
